inc/saitypes.h

```diff
@@ -855,6 +885,19 @@ typedef enum _sai_acl_bind_point_type_t
 
 } sai_acl_bind_point_type_t;
 
+/**
+ * @brief Attribute data for SAI_ACL_TABLE_ATTR_MATCH_TYPE
+ */
+typedef enum _sai_acl_match_type_t
+{
+    /** Match Type Ternary */
+    SAI_ACL_MATCH_TYPE_TERNARY,
+
+    /** Match Type Exact */
+    SAI_ACL_MATCH_TYPE_EXACT
+
```

Comment:
Missing following type:   
 /** Ternary and Exact */
     SAI_ACL_MATCH_TYPE_TERNARY_AND_EXACT,

Reply:
thanks for pointing out. Added

Reply:
What is the difference between TERNARY\_AND\_EXACT and TERNARY? A Ternary type can behave as both exact and ternary. 

What kind of h/w resource would return their type as TERNARY\_AND\_EXACT instead of TERNARY? Also the comments are not adding any additional info.

Reply:
> What is the difference between TERNARY_AND_EXACT and TERNARY? A Ternary type can behave as both exact and ternary. 
> 
> What kind of h/w resource would return their type as TERNARY_AND_EXACT instead of TERNARY? Also the comments are not adding any additional info.

For match type TERNARY_AND_EXACT, the implementation will host some entries in TCAM and some in EM. For example matches that include /120 prefixes will be placed in TCAM while all /128s will be placed in EM. If there is a match in both TCAM and EM, then hardware resolves it by using the EM result. This allows supporting increased scale.

Reply:
As discussed, we need to define the semantics of operations like ACL entry priority, available ACL table entries etc.

Reply:
removed type SAI_ACL_MATCH_TYPE_TERNARY_AND_EXACT based on our discussion. it can be added back when needed with more details on semantics